### DISCUSSION TITLE
spring-boot-2/1-2-quick-start

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#30] [Announcements]
# spring-boot-2/1-2-quick-start

Spring Boot 2.x基础教程：快速入门 简介 在您第1次接触和学习Spring框架的时候，是否因为其繁杂的配置而退却了？在你第n次使用Spring框架的时候，是否觉得一堆反复黏贴的配置有一些厌烦？那么您就不妨来试试使用Spring Boot来让你更易上手，更简单快捷地构建Spring应用！ Spring Boot让我们的Spring应用变的更轻量化。我们不必像以前那样繁琐的构建项目、打包应用、部署到Tomcat等应用服务器中来运行我们的业务服务。通过Spring Boot实现的服务，只需要依靠一个Java类，把它打包成jar，并通过java -jar命令就可以运行起来。这一切相较于传统Spring应用来说，已经变得非常的轻便、简单。

https://www.didispace.com/spring-boot-2/1-2-quick-start.html

**Comment by bianchen955** [comment]
写的非常不错 棒棒哒！
public void getHello() throws Exception {
        mvc.perform(MockMvcRequestBuilders.get("/hello").accept(MediaType.APPLICATION_JSON))
                .andExpect(status().isOk())
                .andExpect(content().string(equalTo("Hello World")));
    }
 .andExpect(content().string(equalTo("Hello World")));    此处类型不匹配，需要的类型是：ResultMatcher  实际类型是：RequestMatcher
